### PR TITLE
unify all message path values

### LINKED ISSUE #802: Unify message paths
As a developer, I want all messages to follow the same naming pattern, so that there is clarity how to construct a path for each message.

The current implementation contains a mixture of various approaches for creating message paths. For example:
- snake case [`cash/update_config`](https://github.com/iov-one/weave/blob/master/x/cash/msg.go#L20)
- camel case [`distribution/resetRevenue`](https://github.com/iov-one/weave/blob/master/x/distribution/msg.go#L17)
- deep path [`gov/electionRules/update`](https://github.com/iov-one/weave/blob/master/x/gov/msg.go#L15)

## Acceptance criteria

- [ ] all messages must follow the same naming pattern `<package>/<message_name>`
- [ ] message path must be snake case
- [ ] `/` must be used only once as the package name separator
- [ ] `<package>` is the package name, for example `escrow` or `cash`
- [ ] `<message_name>` is the message name converted to snake case, for example `CreateRevenueMsg` is `create_revenue`

Some examples:
- `distribution.CreateRevenueMsg` -> `distrubution/create_revenue`
- `gov.UpdateElectionRuleMsg` -> `gov/update_election_rule`

### PR DESCRIPTION
**Pull request by husio**
Unify all message path values to follow the same set of rules.

- all messages must follow the same naming pattern
  `<package>/<message_name>`
- message path must be snake case
- `/` must be used only once as the package name separator
- `<package>` is the package name, for example `escrow` or `cash`
- `<message_name>` is the message name converted to snake case, for
  example `CreateRevenueMsg` is `create_revenue`

resolve #802

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # Changelog
 
 
+Breaking changes
+
+- Unify all message paths to follow pattern `<package>/<message_name>`
```
Comment (ruseinov) [inline CHANGELOG.md]: Curious if it is actually a breaking change. It's not a public api, so it does not break or change anything for the client?

Comment (husio) [reply CHANGELOG.md]: :thinking:

Comment (ethanfrey) [reply CHANGELOG.md]: The only place it is "breaking" is in the genesis configuration of `msgfee`, as well as queries on `msgfee`.

File: cmd/bnsd/x/username/msg.go [hunk 2]
```diff
@@ -25,7 +25,7 @@ func (m *RegisterTokenMsg) Validate() error {
 }
 
 func (RegisterTokenMsg) Path() string {
-	return "username/registerToken"
+	return "username/register_token"
```
Comment (ethanfrey) [inline cmd/bnsd/x/username/msg.go]: :+1:

File: x/aswap/msg.go [hunk 3]
```diff
@@ -16,9 +16,9 @@ func init() {
 }
 
 const (
-	pathCreateSwap   = "swap/create"
-	pathReleaseSwap  = "swap/release"
-	pathReturnReturn = "swap/return"
+	pathCreateSwap  = "aswap/create"
+	pathReleaseSwap = "aswap/release"
+	pathReturnSwap  = "aswap/return"
```
Comment (ethanfrey) [inline x/aswap/msg.go]: :see_no_evil:

File: cmd/bnsd/app/testdata/fixtures/app.go [hunk 4]
```diff
@@ -145,19 +145,19 @@ func appStateGenesis(keyAddress weave.Address) []byte {
 		},
 		"msgfee": []interface{}{
 			dict{
-				"msg_path": "distribution/newrevenue",
```
Comment (ethanfrey) [inline cmd/bnsd/app/testdata/fixtures/app.go]: these definitely look cleaner

**Pull request merged**